packages/odyssey-storybook/src/components/odyssey-mui/Toast/Toast.stories.tsx

```diff
@@ -10,65 +10,62 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
+import { Meta, Story } from "@storybook/react";
+import { Button, Toast, ToastProps, ToastStack } from "@okta/odyssey-react-mui";
 import * as React from "react";
-import { Story } from "@storybook/react";
-import {
-  Alert,
-  AlertTitle,
-  Button,
-  CloseIcon,
-  Link,
-  Snackbar,
-  Stack,
-} from "@okta/odyssey-react-mui";
-import { MuiThemeDecorator } from "../../../../.storybook/components";
 
 import ToastMdx from "./Toast.mdx";
+import { MuiThemeDecorator } from "../../../../.storybook/components";
+import { useState } from "react";
 
-export default {
+const storybookMeta: Meta<ToastProps> = {
   title: `MUI Components/Alerts/Toast`,
-  component: Alert,
+  component: Toast,
   parameters: {
     docs: {
       page: ToastMdx,
     },
   },
   argTypes: {
-    actionLink: {
-      control: "text",
-      default: null,
+    autoHideDuration: {
+      control: "number",
+      defaultValue: null,
+    },
+    isDismissable: {
+      control: "boolean",
     },
-    content: {
+    linkText: {
       control: "text",
-      defaultValue: "Mission to Sirius B scheduled for January 7, 2023",
     },
-    isDismissible: {
-      control: "boolean",
-      defaultValue: null,
+    linkUrl: {
+      control: "text",
     },
     role: {
       control: "radio",
-      options: ["status", null],
+      options: ["alert", "status", null],
       defaultValue: null,
     },
     severity: {
       control: "radio",
       options: ["error", "info", "success", "warning"],
       defaultValue: "info",
     },
+    text: {
+      control: "text",
+      defaultValue: "The mission to Sagittarius A is set for January 7.",
+    },
   },
   decorators: [MuiThemeDecorator],
-};
+} as Meta<ToastProps>;
+
+export default storybookMeta;
 
-const DefaultTemplate: Story = (args) => {
+const DefaultTemplate: Story<ToastProps> = (args) => {
   const [open, setOpen] = React.useState(false);
```

Comment:
I'd like us to use `isOpen` and `setIsOpen` in our stories too if possible.

We could actually use `isVisible` as well since I plan to make us a `useVisibility` hook which will be generic enough for stories like this.